nestkernel/sparse_node_array.cpp

```diff
@@ -24,81 +24,114 @@
 
 // Includes from nestkernel:
 #include "exceptions.h"
+#include "kernel_manager.h"
 #include "node.h"
+#include "vp_manager_impl.h"
+
+
+#include <iomanip>
+#include <iostream>
```

Comment:
Are these two used? I might be overlooking something. If they are used, they should have `// C++ includes:` as a header and be placed over `// Includes from nestkernel:`